plutus-core/cost-model/budgeting-bench/Benchmarks/Unit.hs

```diff
@@ -14,7 +14,7 @@ import System.Random (StdGen)
 
 
 createChooseUnitBench
-    :: (DefaultUni `Includes` a, ExMemoryUsage a, NFData a)
+    :: (DefaultUni `HasTermLevel` a, ExMemoryUsage a, NFData a)
```

Comment:
Bikeshedding: I think "universe 'has term' x" and "universe 'has type' x" are clear enough, not sure the "level" adds anything.

`HasBoth` is confusing, since "both" only makes sense in a context where you know the two things.

Suggestion:
- `HasTerms` -> ```uni `HasTerms` Data``` (`HasTermsOf`?)
- `HasType` -> ```uni `HasType` Data```
- `Includes` (not sure about this, but something that sounds like "fully has this type")

Reply:
> `HasBoth` is confusing, since "both" only makes sense in a context where you know the two things.

Yes, but `HasBothLevel` isn't confusing, because we know what two levels we have.

> `HasTerms`

Meh, it's not terms, it's constants and we already use `HasConstant` for "has a `Constant`-like constructor, so adding a completely different `HasConstants` would be confusing. While ``uni `HasTermLevel` Integer`` does read (albeit a bit weirdly) like you can use an `Integer` at the term level when dealing with a term having access to the `uni` universe.

> `HasType`

This is way too abstract of a name to be stolen by the builtins machinery for very specific usage, I feel.